app/views/games/show.html.erb

```diff
@@ -47,7 +47,7 @@
 <div class="game-wrapper" data-current-turn="<%= @game.current_player_turn %>" data-player="<%= get_player %>" data-black-time-left="<%= @black_player_timer.time_left %>" data-white-time-left="<%= @white_player_timer.time_left %>" data-game-id="<%= @game.id %>" data-time-now="<%= Time.now.to_i %>" data-white-start-time="<%= @white_player_timer.start_time.to_i %>" data-black-start-time="<%= @black_player_timer.start_time.to_i %>" data-game-type="<%= @game.is_blitz %>" data-has-started="<%= @game.has_started %>">
   <div class="white-player">White Player's Time Left:<div id="white-player-timer"></div></div>
   <% else %>
-  <div class="game-wrapper" data-current-turn="<%= @game.current_player_turn %>">
+  <div class="game-wrapper" data-current-turn="<%= @game.current_player_turn %>" data-has-started="<%= @game.has_started %>>
```

Comment:
I think you're missing a quote right before the ending >. (Just taking this fix for my branch as well, thanks!)